assets/src/components/ui/dates-and-times/calendar-page-date/index.js

```diff
@@ -116,27 +109,31 @@ class CalendarPageDate extends Component {
 		);
 	};
 
-	render() {
-		const {
-			startDate = null,
-			endDate = null,
-			size = 'small',
-			statusClass = '',
-			...otherProps
-		} = this.props;
-		const htmlClass = classNames( {
-			'ee-calendar-page-date-wrapper': true,
-			[ `ee-calendar-page-date-${ size }` ]: true,
+	const htmlClass = classNames( {
+		'ee-calendar-page-date-wrapper': true,
+		[ `ee-calendar-page-date-${ size }` ]: true,
```

Comment:
don't know why I did this as an object 🤷‍♂ 
would have made more sense to just list the classes...
if if you end up making other changes to this branch, then maybe fix this too